src/App.jsx

```
@@ -24,6 +24,15 @@ function App() {
     messagesEndRef.current?.scrollIntoView({ behavior: 'smooth' })
   }
 
+  // Load saved username from localStorage on app startup
+  useEffect(() => {
+    const savedUsername = localStorage.getItem('chatapp-username')
+    if (savedUsername && savedUsername.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)